tests/internal/remoteconfig/test_remoteconfig_client.py

```diff
@@ -657,3 +662,27 @@ def test_remote_config_client_steps(mock_appsec_rc_capabilities, mock_send_reque
         mock_callback.assert_not_called()
         mock_send_request.reset_mock()
         mock_callback.reset_mock()
+
+
+@pytest.mark.subprocess(env={"DD_TAGS": "env:foo,version:bar"})
+def test_remote_config_client_tags():
+
+    from ddtrace.internal.remoteconfig.client import RemoteConfigClient
+
+    tags = dict(_.split(":", 1) for _ in RemoteConfigClient()._client_tracer["tags"])
+
+    assert tags["env"] == "foo"
+    assert tags["version"] == "bar"
+
+
+@pytest.mark.subprocess(
+    env={"DD_TAGS": "env:foooverridden,version:baroverridden", "DD_ENV": "foo", "DD_VERSION": "bar"}
```

Comment:
Out of curiosity, how do we parse out the `overridden` from a tag set in `DD_TAGS` like here?